service/evidence/evidence_store_test.go

```diff
@@ -293,34 +297,234 @@ func TestService_StoreEvidences(t *testing.T) {
 
 // TestListEvidences tests List req
 func TestService_ListEvidences(t *testing.T) {
-	// TODO(oxisto): Convert this test to a table test
-	s := NewService()
-	err := s.storage.Create(&evidence.Evidence{
-		Id:             testdata.MockEvidenceID,
-		CloudServiceId: testdata.MockCloudServiceID,
-		Timestamp:      timestamppb.Now(),
-		Raw:            util.Ref(""),
-		Resource:       structpb.NewNullValue(),
-	})
-	assert.NoError(t, err)
-	err = s.storage.Create(&evidence.Evidence{
-		Id:             testdata.MockAnotherEvidenceID,
-		CloudServiceId: testdata.MockAnotherCloudServiceID,
-		Timestamp:      timestamppb.Now(),
-		Raw:            util.Ref(""),
-		Resource:       structpb.NewNullValue(),
-	})
-	assert.NoError(t, err)
+	type fields struct {
+		storage persistence.Storage
+		authz   service.AuthorizationStrategy
+	}
+	type args struct {
+		in0 context.Context
+		req *evidence.ListEvidencesRequest
+	}
+	tests := []struct {
+		name     string
+		fields   fields
+		args     args
+		wantResp assert.ValueAssertionFunc
+		wantErr  assert.ErrorAssertionFunc
+	}{
+		{
+			name: "Successful Filter Of Evidences (with allowed cloud service)",
+			fields: fields{
+				authz: newAuthorizationStrategy([]string{evidencetest.MockEvidence1.CloudServiceId}),
+				storage: testutil.NewInMemoryStorage(t, func(s persistence.Storage) {
+					assert.NoError(t, s.Create(&evidencetest.MockEvidence1))
+					assert.NoError(t, s.Create(&evidencetest.MockEvidence2))
+				}),
+			},
+			args: args{
+				in0: context.TODO(),
+				req: &evidence.ListEvidencesRequest{
+					PageSize:  evidencetest.MockListEvidenceRequest1.PageSize,
+					PageToken: evidencetest.MockListEvidenceRequest1.PageToken,
+					OrderBy:   evidencetest.MockListEvidenceRequest1.OrderBy,
+					Asc:       evidencetest.MockListEvidenceRequest1.Asc,
+					Filter: &evidence.Filter{
+						CloudServiceId: evidencetest.MockListEvidenceRequest1.Filter.CloudServiceId,
+						ToolId:         evidencetest.MockListEvidenceRequest1.Filter.ToolId,
+					},
+				},
```

Comment:
Ah, I just recognize it: You can simply use `evidencetest.MockListEvidenceRequest1` as request (`req: evidencetest.MockListEvidenceRequest1`). Or do I miss something?